AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginAuthIntegrationTests/README.md

```diff
@@ -0,0 +1,74 @@
+## DataStore with Auth Integration Tests
+
+The following steps demonstrate how to set up an GraphQL endpoint with AppSync. 
+The auth configured will be Cognito UserPools. 
+This set up is used to run the tests in `AWSDataStoreCategoryPluginAuthIntegrationTests.swift`.
+
+### Set-up
+
+1. `amplify init`
+
+2. `amplify add api`
+
+```perl
+? Please select from one of the below mentioned services: GraphQL
+? Provide API name: `<APIName>`
+? Choose the default authorization type for the API `Amazon Cognito User Pool`
+? Do you want to use the default authentication and security configuration? `Default configuration`
+? How do you want users to be able to sign in? `Email`
+? Do you want to configure advanced settings? `No, I am done.`
+? Do you want to configure advanced settings for the GraphQL API `No, I am done.`
+? Do you have an annotated GraphQL schema? `No`
+? Do you want a guided schema creation? `Yes`
+? Do you want to edit the schema now? `No`
+```
+
+The guided schema provided should look like this: 
+```json
+type SocialNote
+    @model
+    @auth(rules: [
+        { allow: owner, ownerField: "owner", operations: [create, update, delete] },
+    ]) {
+    id: ID!
+    content: String!
+    owner: String
+}
+```
+
+3. `amplify update api`
+? Please select from one of the below mentioned services: `GraphQL`
+? Select from the options below `Enable DataStore for entire API`
+
+4. `amplify push`
+```perl
+? Are you sure you want to continue? `Yes`
+? Do you want to generate code for your newly created GraphQL API `No`
+```
+
+5. Copy `amplifyconfiguration.json` over as `AWSDataStoreCategoryPluginAuthIntegrationTests-amplifyconfiguration.json`
```

Comment:
Are you saying to change the file name from amplifyconfiguration.json to AWSDataStoreCategoryPluginAuthIntegrationTests-amplifyconfiguration.json ?

Reply:
yes, currently they're added to the HostApp's bundle. If we have multiple integration test targets, we need to prefix the configuration so each target reads their own configuration file

Reply:
I see, we can word this to:
Copy `amplifyconfiguration.json` to a new file named `AWSDataStoreCategoryPluginAuthIntegrationTests-amplifyconfiguration.json`